### ISSUE TITLE
Comments aren't easy to access on mobile

### ISSUE THREAD

**Issue by RedRiderX** [enthusiast94/reddit-reader-backbone#3]
Right now there isn't a very good way to access the comments for a link on mobile.

Maybe add a button for comments? Or hide it behind a long tap?

**Comment by enthusiast94** [comment]
Yeah, probably a button would be nice. Will look into it soon. Thanks!
